sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserRightsFacadeEjb.java

```diff
@@ -189,8 +189,7 @@ private void generateUserRightsDocument(Map<UserRoleDto, Set<UserRight>> userRol
 			columnIndex = 3;
 			for (UserRoleDto userRole : userRoleRights.keySet()) {
 				Cell roleRightCell = row.createCell(columnIndex);
-				if (userRoleRights.containsKey(userRole) && userRoleRights.get(userRole).contains(userRight)
-					|| userRoleFacade.hasUserRight(Collections.singletonList(userRole), userRight)) {
+				if (userRoleFacade.hasUserRight(Collections.singletonList(userRole), userRight)) {
```

Comment:
1. `userRoleRights.containsKey(userRole)` is always `true` because `userRole` comes from that Map.
2. `userRoleRights.get(userRole).contains(userRight)` and `userRoleFacade.hasUserRight(Collections.singletonList(userRole), userRight)` are equal because the Facade method does internally the same check.